### DISCUSSION TITLE
Welcome, Rfk!

### DISCUSSION THREAD

**Discussion by Rfkgaming89** [ParanoidUser/yolo#614] [Subscribe]
would love to contribute

**Comment by ParanoidUser** [comment]
Ahoy @Rfkgaming89, welcome aboard!

The invitation to join the collaborator group has been sent to you. Thanks for your interest in this project!
Feel free to post your own questions to the community, or share your path to earning other badges. 😊 Let me know if you have any questions.

Cheers
